src/acceptance-test/java/org/zalando/nakadi/webservice/hila/HilaAT.java

```diff
@@ -399,4 +402,41 @@ public void whenStreamTimeout0ThenInfiniteStreaming() throws Exception {
                 .anyMatch(streamBatch -> streamBatch.getMetadata() != null
                         && streamBatch.getMetadata().getDebug().equals("Stream timeout reached")));
     }
+
+    @Test(timeout = 15000)
+    public void whenResetCursorsThenStreamFromResetCursorOffset() throws Exception {
```

Comment:
@adyach no, wait. I'm wrong. "stream started" should come with a first batch actually. So there should be only 10. Ok , I will check deeply why it fails for me.